### PR TITLE
Bitcoin 0.20.1

### PR DESCRIPTION
**Pull request by luke-jr**
cc @juippis @candrews @thesamesam

### CODE REVIEW

**Comment by candrews** [comment]
`libressl` was dropped - was that intentional?

**Comment by luke-jr** [comment]
It should have been dropped in 0.20.0; we no longer depend on OpenSSL.

File: net-p2p/bitcoin-qt/bitcoin-qt-0.20.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+IUSE="+asm dbus kde knots +qrcode +system-leveldb test upnp +wallet zeromq"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/boost-1.52.0:=[threads(+)]
+	>dev-libs/libsecp256k1-0.1_pre20170321:=[recovery]
+	>=dev-libs/univalue-1.0.4:=
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
```
Comment (thesamesam) [inline net-p2p/bitcoin-qt/bitcoin-qt-0.20.1.ebuild:33]: No := on any of these? I assume they're linked against?

File: net-p2p/bitcoin-cli/bitcoin-cli-0.20.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+IUSE="knots"
+
+DEPEND="
+	>=dev-libs/boost-1.52.0:=[threads(+)]
+	dev-libs/libevent:=
+	>=dev-libs/univalue-1.0.4:=
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	>=sys-devel/autoconf-2.69
```
Comment (thesamesam) [inline net-p2p/bitcoin-cli/bitcoin-cli-0.20.1.ebuild:31]: Use autotools' variables

File: dev-util/bitcoin-tx/bitcoin-tx-0.20.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+	rm -r src/leveldb src/secp256k1 || die
+}
+
+src_configure() {
+	local my_econf=(
+		--disable-asm
+		--without-qtdbus
+		--without-qrencode
+		--without-miniupnpc
+		--disable-tests
```
Comment (thesamesam) [inline dev-util/bitcoin-tx/bitcoin-tx-0.20.1.ebuild:81]: The tests are just in the other packages, right?

**Pull request closed without merging**